.gitignore

```diff
@@ -108,6 +108,7 @@ env/
 venv/
 cfvenv/
 afvenv*/
+lenv*/
 ENV/
```

Comment:
Here i just added another virtualenvironmewnt for wsl in my local machine and added this so it does not ask me all the time to add these files